fdm-core/src/soil.ts

```
@@ -2,15 +1,18 @@
 import * as schema from "./db/schema"
 import { handleError } from "./error"
 import type { FdmType } from "./fdm"
 import { createId } from "./id"
+import type { Timeframe } from "./timeframe"
 import type {
     CurrentSoilData,
     getSoilAnalysisType,
     SoilParameterDescription,
     SoilParameters,
 } from "./soil.d"
+import type { getSoilAnalysisType } from "./soil.d"
```

Biome (1.9.4)

[error] Shouldn't redeclare 'getSoilAnalysisType'. Consider to delete it or rename it.

'getSoilAnalysisType' is defined here:


(lint/suspicious/noRedeclare)